test/parallel/test-child-process-chunked-stdout.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+const stream = require('stream');
+const path = require('path');
+
+const producer = fork(path.join(common.fixturesDir, 'stdout-producer.js'),
+    { silent: true });
```

Comment:
@bnoordhuis I'm not aware of a linter rule like this. Did closure-linter have that feature?